learning-progression-with-la-schema.md

```diff
@@ -1058,6 +1058,10 @@ Program state. List with examples.
 ##### Clock cycles
```

Comment:
For the "To-Do" on Instruction Set Architecture:

Should I start illustrating this with F4-MISC instructions? This is before we have really introduced assembly, so I'm not sure how to go about this. I think I would want to explain this with a diagram of a basic CPU. First, explaining the translation of instructions to binary. Then, moving to how the instruction moves through the hardware, similar to concepts covered in Computer Org. I.